src/test/resources/pages/instructorStudentRecordsPageWithPrivateFeedback.html

```diff
@@ -128,7 +128,7 @@ <h4 class="modal-title">
     <div class="row">
       <div class="col-md-12">
         <br>
-        <div class="student_feedback panel panel-default" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
+        <div class="student_feedback panel panel-default load-feedback-session" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
```

Comment:
I'd suggest reverting the entire file first, then re-running `InstructorStudentRecordsPageUiTest` with GodMode enabled. The only lines that should change are those containing links to `index.jsp`. Any other unrelated change indicates a problem with your config somewhere - perhaps the issue lies elsewhere, but do merge the latest `master` and do `npm run build` again before re-running the test just in case.